src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -694,7 +694,9 @@ public IEnumerable<string> GetCollections(Transaction transaction)
 
                         hasValue = true;
 
-                        using (Slice.From(context.Allocator, revision.ChangeVector, out var keySlice))
+                        using (Slice.From(context.Allocator, revision.ChangeVector, out var changeVectorSlice))
+                        using (context.Allocator.Allocate(prefixSlice.Size + changeVectorSlice.Size, out var keyBuffer))
```

Comment:
the prefix here is the id of the document?

Reply:
let move this all inside the `CreateRevisionTombstoneKeySlice`

Reply:
Done